src/lib/localisation.ts

```diff
@@ -0,0 +1,60 @@
+/*
+	Date: April 2021
+	This is a reimplementation in DCR, for parity of the Localisation.scala file in frontend.
+	I realise that the replacement items are rather adhoc but again, we are just ensuring parity.
+*/
+
+// How to read the repacement instructions ?
+// ["US", "Film", "Movies"] , means: If the edition is US, then replace "Film" by "Movies"
+// Instructions in the caseInsensitive array must also work in lowecase, therefore replace "film" by "movies"
+
+type TranslationInstruction = [string, string, string];
+
+const caseInsensitive: TranslationInstruction[] = [
+	['US', 'Film', 'Movies'],
+	['US', 'Football', 'Soccer'],
+];
+
+const caseSensitive: TranslationInstruction[] = [
+	['US', 'in film', 'in movies'],
+	['US', 'in football', 'in soccer'],
+	[
+		'US',
+		'Football news, match reports and fixtures',
+		'Soccer news, match reports and fixtures',
+	],
+];
```

Comment:
Do we need two lists? Could you perhaps have one big list that contains the case variations and then you do case sensitive comparisons?

Not sure how that would work with the code but it seems like it might have a better developer experience in terms of adding more examples in future